packages/firestore/test/integration/browser/webchannel.test.ts

```diff
@@ -20,6 +20,11 @@ import { DatabaseId, DatabaseInfo } from '../../../src/core/database_info';
 import * as utilHelpers from '../util/helpers';
 
 describe('WebChannel', () => {
```

Comment:
Can we do something like: `(typedef window | describe ? xdescribe)('WebChannel', ...`